src/scrape_up/github/users.py

```diff
@@ -184,53 +196,3 @@ def get_starred_repos(self):
         except:
             message = f"Starred repositories not found for username {self.username}"
             return message
-    
-    def __scrape_followers_page(self):
-        """
-        Scrape the followers page of a GitHub user.
-        """
-        username = self.username
-        followers_data = requests.get(f"https://github.com/{username}?tab=followers")
-        followers_data = BeautifulSoup(followers_data.text, "html.parser")
-        return followers_data
-
-    def get_followers(self):
-        """
-        Fetches the following users of a GitHub user.
-        """
-        page = self.__scrape_followers_page()
-        try:
-            followers_body = page.find('turbo-frame', id = 'user-profile-frame')
-            followers = []
-            for user in followers_body.find_all('span', class_='Link--secondary'):
-                followers.append(user.text.strip())
-            
-            return followers
-        except:
-            message = f"Followers not found for username {self.username}"
-            return message
-    
-    def __scrape_following_page(self):
-        """
```

Comment:
Why you deleted all these methods? @codesleep-Beperfect

Reply:
it was deleted while resolving merges
@nikhil25803 should i recreate the pull request?

Reply:
That would be better, create a new PR.

Reply:
Okay Cool